karspexet/ticket/tests/test_models.py

```diff
@@ -41,14 +45,69 @@ def test_finalized_reservation_is_active(self):
         self.assertTrue(reservation in Reservation.active.all())
 
     def test_timed_out_reservation_is_not_active(self):
-        timestamp = datetime.now() - relativedelta(days=1)
+        timestamp = timezone.now() - relativedelta(days=1)
         reservation = Reservation.objects.create(
             show=self.show,
             session_timeout=timestamp,
             tickets={},
             total=200,
         )
 
-
         self.assertTrue(reservation in Reservation.objects.all())
         self.assertFalse(reservation in Reservation.active.all())
+
+
+    def test_total_price(self):
+        timestamp = timezone.now() - relativedelta(days=1)
+        reservation = Reservation.objects.create(
+            show=self.show,
+            total=0,
+            session_timeout=timestamp,
+            tickets={
+                self.seat1.id: 'student',
+                self.seat2.id: 'normal',
+            },
+        )
+
+        self.assertEqual(reservation.total_price(), 450)
```

Comment:
We're using pytest, right? Then you should just use `assert <boolean expression>`, to let it pretty print the failure.